### PR TITLE
Add README header validation for SDKs

### PR DESCRIPTION
**Pull request by cprice404**
This extends the README generator so that it can do some validation on the list of headers that appear in the README for an SDK.  The goal is to allow us to enforce that all of the SDKs have some level of consistency in their section headers.

I'm sure this will evolve more over time, but this was my best guess at a good starting point.

### CODE REVIEW

**Comment by cprice404** [comment]
@eaddingtonwhite @malandis this is ready for review now.  I think it's good enough for a first stab at this.

@bruuuuuuuce you may also have thoughts on it 🤷

File: github-actions/oss-readme-template/__tests__/readme.test.ts [hunk 1]
```diff
@@ -52,11 +67,170 @@ STATIC CONTENT
...
+        projectStatus: ProjectStatus.OFFICIAL,
+        projectStability: ProjectStability.STABLE,
+      })
+    )
+      .toEqual(`<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+[![project status](https://momentohq.github.io/standards-and-practices/badges/project-status-official.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+[![project stability](https://momentohq.github.io/standards-and-practices/badges/project-stability-stable.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md) 
+
+# Momento WaterLoop Client Library
```
Comment (eaddingtonwhite) [inline github-actions/oss-readme-template/__tests__/readme.test.ts:103]: `WaterLoop`? Sorry I missed this before maybe what is this exactly? Never heard term before.

Comment (eaddingtonwhite) [reply github-actions/oss-readme-template/__tests__/readme.test.ts:103]: Is it just supposed to be a hypothetical lang?

File: github-actions/oss-readme-template/src/markdownlint-rules/verify-sdk-section-headers.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  information: new URL(
+    'https://github.com/momentohq/standards-and-practices/github-actions/oss-readme-generator'
+  ),
+  function: (params, onError) => {
+    const headers: Array<HeaderWithLineNumber> = [];
+    let insideHeaderOpenTag = false;
+    let headerOpenLineNumber = 1;
+    let currentContent: string | undefined = undefined;
+    for (const token of params.tokens) {
+      // console.log(`TYPE: '${token.type}'`);
```
Comment (eaddingtonwhite) [inline github-actions/oss-readme-template/src/markdownlint-rules/verify-sdk-section-headers.ts:25]: mean to remove?

**Review by eaddingtonwhite — APPROVED** [review]
Awesome looks great! Nice work on tests as well this is going to help aton managing all the repos 💯

File: github-actions/oss-readme-template/src/markdownlint-rules/verify-sdk-section-headers.ts (hunk 2 above)
Comment (cprice404) [reply github-actions/oss-readme-template/src/markdownlint-rules/verify-sdk-section-headers.ts:25]: ah yeah thx

**Review by eaddingtonwhite — APPROVED** [review]

**Review by pgautier404 — APPROVED** [review]

**Pull request merged**